docs/documentation/gui.md

```diff
@@ -71,7 +71,7 @@ To use the GUI, start by opening the app, select the form corresponding to the a
 1. **Field name**: Indicates the name or label of the field.
 2. **File explorer button**: Allows users to browse and select a file (available for path-related fields, **excluding the web version**).
 3. **Default button**: Clicking on this button, if colored, automatically adds a default value to the field.
-4. **Information icon**: When hovered over with the mouse, provides a description of the field.
+4. **Information icon**: When hovered over with the mouse, it describes the field.
 5. **Required icon**: If colored, indicates that the field must be filled out and cannot be left empty.
 
 !!!Warning
```

Comment:
If this is referring to the node.js local web app, that's another reason to remove it.